### ISSUE TITLE
Off hand "Hand Model" + Skin

### ISSUE THREAD

**Issue by CJCrafter** [WeaponMechanics/WeaponMechanicsCosmeticsWiki#2]
### Description

When a player is holding a weapon, a packet based (fake) item should be put in the player's offhand. This would be a model that could appear to be physically holding the weapon. The hand model should be able to use the [WeaponMechanics Skin System](https://github.com/WeaponMechanics/MechanicsMain/wiki/Skins) which supports sprinting, reloading, etc.

We should also allow different color hands.

Paid for by: `ASTRICK#1775`

### Configuration

We would have a folder called `server > plugins >WeaponMechanicsCosmetics > Hands` which has:
```yaml
Hand_AK-47:
  Base_Item: <ItemSerializer> # this can be used to give the item a name/lore/whatever
  Skin: <SkinSerializer> # used WeaponMechanics skin to change based on sprint/reload/whatever
```

Then in the `server > plugins > WeaponMechanics > weapons > assault_rifles > ak-47.yml` file, we would have
```yaml
AK-47:
  Cosmetics:
    Off_Hand_Hand: "Hand_AK-47"
```

**Comment by CJCrafter** [comment]
Before this can be implemented, we have to wait for the skins feature to be updated in WeaponMechanics and for the [SQL stuff](https://github.com/WeaponMechanics/MechanicsMain/pull/155) to be merged.

**Comment by CJCrafter** [comment]
Added basic skin support, working on fake items:
[0a51a88](https://github.com/WeaponMechanics/MechanicsMain/commit/0a51a882d9614daa257efd59085f1cf1fe457e33)

**Comment by CJCrafter** [comment]
For VR players, the off hand should be completely disabled. We can set a toggle for this in the `config.yml`
```yaml
Disable_Hand_In_VR: <true/false>
```

**Comment by CJCrafter** [comment]
Implemented in `1.2.0`.

Bugs:
* Need to setup permissions
* Fix sprint skin applying on a delay

**Comment by CJCrafter** [comment]
Need to disable feature in creative mode

**Comment by CJCrafter** [comment]
Completely implemented in v1.2.2
